projects/bactteria_detection/datasets/bactteria_detection_dataset.py

```diff
@@ -0,0 +1,28 @@
+from mmseg.datasets import BaseSegDataset
+from mmseg.registry import DATASETS
+
+
+@DATASETS.register_module()
+class BactteriaDetectionDataset(BaseSegDataset):
+    """BactteriaDetectionDataset dataset.
+
+    In segmentation map annotation for BactteriaDetectionDataset,
+    0 stands for background, which is included in 3 categories.
+    ``reduce_zero_label`` is fixed to False. The ``img_suffix``
+    is fixed to '.png' and ``seg_map_suffix`` is fixed to '.png'.
+    Args:
+        img_suffix (str): Suffix of images. Default: '.png'
+        seg_map_suffix (str): Suffix of segmentation maps. Default: '.png'
```

Comment:
Please add `reduce_zero_label` in docstring, as people need to know this arg's work